Formula/b/basti.rb

```diff
@@ -0,0 +1,31 @@
+require "language/node"
+
+class Basti < Formula
+  desc "Cloud credentials manager cli"
+  homepage "https://www.basti.app/"
+  url "https://registry.npmjs.org/basti/-/basti-1.6.2.tgz"
+  sha256 "8d813c1f4e3b8195655d40e670aa8a2eb7dce2cd21d996564c56a3296163f1d7"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    # Remove incompatible pre-built binaries
+    os = OS.kernel_name.downcase
+    arch = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s
+    node_modules = libexec/"lib/node_modules/basti/node_modules"
+    rm_f node_modules/"basti-session-manager-binary-#{os}-#{arch}/session-manager-plugin"
+
+    generate_completions_from_executable(bin/"basti", "completion")
```

Comment:
This only seems to work for bash and not for zsh. The error thrown is because it tries to run `basti completion zsh` which returns 

```
~ basti completion zsh
basti <command>

Commands:
  basti init                  Initialize a target to use with Basti                                                                                [aliases: i]
  basti connect [connection]  Start port forwarding session with the selected target                                                               [aliases: c]
  basti cleanup               Remove all resources created by Basti                                                                               [aliases: cl]
  basti completion            Generate completion script for your shell

Options:
  -h, --help     Show help                                                                                                                            [boolean]
  -v, --version  Show version number                                                                                                                  [boolean]

Examples:
  basti init     Initialize a target in interactive mode
  basti connect  Start connection in interactive mode
  basti cleanup  Cleanup Basti resources in interactive mode (requires confirmation)

Did you mean cl?
➜  ~ echo $?
1
```

Reply:
Maybe for now we can remove this line?

Reply:
yeah, we should also raise an upstream issue about this (not sure if this is yargs version issue or not)